bitnami/mariadb-galera/templates/statefulset.yaml

```diff
@@ -128,6 +128,16 @@ spec:
             - name: MARIADB_EXTRA_FLAGS
               value: {{ .Values.extraFlags | quote }}
             {{- end }}
+            {{- if .Values.ssl.rt.enabled }}
+            - name:  MARIADB_ENABLE_SSL_RT
+              value: yes
```

Comment:
You might want to move this environment variable before the `{{- if .Values.ssl.rt.enabled }}`, and use:
```
            - name:  MARIADB_ENABLE_SSL_RT
              value: {{ ternary "yes" "no" .Values.ssl.rt.enabled | quote }}
```
This way you're always setting the env. variable and you don't need to rely on the default value that comes from the container.